assets/js/modules/adsense/dashboard/adsense-module-status.js

```diff
@@ -118,13 +119,15 @@ const propsFromAccountStatus = ( accountStatus, existingTag ) => {
 			};
 			break;
 		case 'ads-display-pending':
-			statusHeadline = __( 'We’re getting your site ready for ads', 'google-site-kit' );
+			const parsedDomain = pslParse( siteURL.hostname );
```

Comment:
This might be a bit cleaner if we just take the part we care about so we don't end up with `parsedDomain.domain` later.

```suggestion
			const { domain: parsedDomain } = pslParse( siteURL.hostname );
```